README.md

```diff
@@ -1,12 +1,12 @@
-# Shopsys Project-base
+# Shopsys Framework Demoshop
 
-This repository is part of Shopsys Framework - a **scalable ecommerce framework** for fast-growing ecommerce sites created and maintained by in-house developers or outsourcing agencies.
+This repository is Demoshop project of Shopsys Framework - a **scalable ecommerce framework** for fast-growing ecommerce sites created and maintained by in-house developers or outsourcing agencies.
 
-Shopsys/project-base repository contains the fully ready base for building your Shopsys Framework projects. This repository is READ-ONLY and it is maintained by monorepo [shopsys/shopsys](https://github.com/shopsys/shopsys).
+Shopsys/demoshop repository contains demostration project to show how you can build your own project on top of Shopsys Framework. It started on July of 2018 based on [shopsys/project-base](https://github.com/shopsys/project-base) and since then contains several new implementational features. Its purpose is to demonstrate possibilities of Shopsys Framework and also to test and validate all concepts of Shopsys Framewrk to ensure customization and upgradability.
```

Comment:
Please, put every sentence on a separate line, it will result in readable diffs and it will be maintained more easily in the future.

Reply:
The are some typos: "...contains *a* demostration...", "...started *in* July...", "...demonstrate *the* possibilities...", "...ensure *customizability* and..."

Reply:
the wording "since then contains several new implementational features" seems wrong to me